jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/common/Chip.java

```diff
@@ -12,7 +15,7 @@
  * To see an example of Chip web element please visit https://vuetifyjs.com/en/components/chips/
  */
 
-public class Chip extends UIBaseElement<ChipAssert> {
+public class Chip extends UIBaseElement<ChipAssert> implements HasClick, HasIcon, HasLabel {
```

Comment:
HasClick is not implemented, in case of removable type we should be sure that random click will not click on remove button, so we should click on label or another "safe" part.